autogluon/training/docker/0.2/py3/Dockerfile.cpu

```diff
@@ -55,7 +56,17 @@ RUN apt-get update \
  && apt-get autoremove -y \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
- 
+
+# install OpenSSL
+RUN wget -q -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
```

Comment:
Is there a reason we can't install openSSL from apt-get?

Reply:
ditto